src/xo-app/xoa-updates/index.js

```diff
@@ -19,8 +19,14 @@ import { serverVersion } from 'xo'
 
 import pkg from '../../../package'
 
+const promptForReload = () => confirm({
+  title: _('promptUpgradeReloadTitle'),
+  body: <p>{_('promptUpgradeReloadMessage')}</p>
+}).then(() => window.location.reload())
+
 if (+process.env.XOA_PLAN < 5) {
   xoaUpdater.start()
+  xoaUpdater.on('promptForReload', promptForReload)
```

Comment:
Looks weird to me: `xoaUpdater` should not trigger a `promptForReload` but a `updateSuccessful` event.